### PR TITLE
Update for outer join reorder

### PR DESCRIPTION
**Pull request by chrysan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: join-reorder.md [hunk 1]
```diff
@@ -30,5 +30,6 @@ aliases: ['/docs-cn/dev/join-reorder/','/docs-cn/dev/reference/performance/join-
 当前的 Join Reorder 算法存在如下限制：
 
 - 受结果集的计算算法所限并不会保证一定会选到合适的 Join order
+- 目前默认不支持 Outer Join 的 Join Reorder，需要设置系统变量 [`tidb_enable_outer_join_reorder`](/system-variables.md#tidb_enable_outer_join_reorder-从-v610-版本开始引入) 的值为 `ON` 开启
```
Comment (shichun-0415) [inline join-reorder.md:33]:
```suggestion
- 目前默认不支持 Outer Join 的 Join Reorder。如需开启此功能，需要将系统变量 [`tidb_enable_outer_join_reorder`](/system-variables.md#tidb_enable_outer_join_reorder-从-v610-版本开始引入) 的值设为 `ON`。
```

File: system-variables.md [hunk 2]
```diff
@@ -920,9 +920,9 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
-- 默认值：`ON`
-- 自 v6.1.0 起，TiDB 的 [Join Reorder 算法](/join-reorder.md) 开始支持 Outer Join。该变量用于控制这个支持行为，默认开启。
-- 对于从旧版本升级上来的集群，该变量的默认值也会是 `TRUE`。
+- 默认值：`OFF`
+- 自 v6.1.0 起，TiDB 的 [Join Reorder 算法](/join-reorder.md) 开始支持 Outer Join。该变量用于控制这个支持行为，默认关闭。
```
Comment (shichun-0415) [inline system-variables.md:924]:
```suggestion
- 自 v6.1.0 起，TiDB 的 [Join Reorder 算法](/join-reorder.md) 开始支持 Outer Join。该变量用于控制这个支持行为，默认关闭，即不启用 Outer Join 的 Join Reorder。
```

File: system-variables.md [hunk 3]
```diff
@@ -920,9 +920,9 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
+- 对于从旧版本升级上来的集群，该变量的默认值也会是 `FALSE`。
```
Comment (shichun-0415) [inline system-variables.md:925]: 前面默认值是 `OFF`，这里是 `FALSE`，为什么不一样呢？
```suggestion
- 对于从旧版本升级上来的集群，该变量的默认值也会是 `FALSE`。
```

Comment (TomShawn) [reply system-variables.md:925]: @chrysan 这里是说升级后保持原版本的默认值吗？

File: join-reorder.md (hunk 1 above)
Comment (TomShawn) [reply join-reorder.md:33]: 既然都能手动开启了，是不是默认关闭呢？

File: system-variables.md [hunk 4]
```diff
@@ -920,9 +920,9 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
-- 默认值：`ON`
-- 自 v6.1.0 起，TiDB 的 [Join Reorder 算法](/join-reorder.md) 开始支持 Outer Join。该变量用于控制这个支持行为，默认开启。
-- 对于从旧版本升级上来的集群，该变量的默认值也会是 `TRUE`。
+- 默认值：`OFF`
+- 自 v6.1.0 起，TiDB 的 [Join Reorder 算法](/join-reorder.md) 开始支持 Outer Join。该变量用于控制这个支持行为，默认关闭，即不启用 Outer Join 的 Join Reorder。
+- 对于从旧版本升级上来的集群，该变量的默认值也会是 `FALSE`。
```
Comment (chrysan) [inline system-variables.md:925]:
```suggestion
- 对于从旧版本升级上来的集群，该变量的默认值也会是 `OFF`。
```

File: join-reorder.md [hunk 5]
```diff
@@ -30,5 +30,6 @@ aliases: ['/docs-cn/dev/join-reorder/','/docs-cn/dev/reference/performance/join-
 当前的 Join Reorder 算法存在如下限制：
 
 - 受结果集的计算算法所限并不会保证一定会选到合适的 Join order
+- 目前默认不支持 Outer Join 的 Join Reorder。如需开启此功能，需要将系统变量 [`tidb_enable_outer_join_reorder`](/system-variables.md#tidb_enable_outer_join_reorder-从-v610-版本开始引入) 设为 `ON`。
```
Comment (TomShawn) [inline join-reorder.md:33]:
```suggestion
- 目前默认关闭 Outer Join 的 Join Reorder。如需开启此功能，需要将系统变量 [`tidb_enable_outer_join_reorder`](/system-variables.md#tidb_enable_outer_join_reorder-从-v610-版本开始引入) 设为 `ON`。
```

File: system-variables.md [hunk 6]
```diff
@@ -920,9 +920,9 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
-- 默认值：`ON`
-- 自 v6.1.0 起，TiDB 的 [Join Reorder 算法](/join-reorder.md) 开始支持 Outer Join。该变量用于控制这个支持行为，默认开启。
-- 对于从旧版本升级上来的集群，该变量的默认值也会是 `TRUE`。
+- 默认值：`OFF`
+- 自 v6.1.0 起，TiDB 的 [Join Reorder 算法](/join-reorder.md) 开始支持 Outer Join。该变量用于控制这个支持行为，默认关闭，即不启用 Outer Join 的 Join Reorder。
```
Comment (TomShawn) [inline system-variables.md:924]:
```suggestion
- 自 v6.1.0 起，TiDB 的 [Join Reorder 算法](/join-reorder.md) 开始支持 Outer Join。该变量用于控制这个支持行为。
```

File: system-variables.md [hunk 7]
```diff
@@ -920,9 +920,9 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
-- 默认值：`ON`
-- 自 v6.1.0 起，TiDB 的 [Join Reorder 算法](/join-reorder.md) 开始支持 Outer Join。该变量用于控制这个支持行为，默认开启。
-- 对于从旧版本升级上来的集群，该变量的默认值也会是 `TRUE`。
+- 默认值：`OFF`
```
Comment (TomShawn) [inline system-variables.md:923]:
```suggestion
- 默认值：在 v6.1.0 中为 `ON`，即默认开启。在 v6.1.0 之后的版本中为 `OFF`，即默认关闭。
```

File: system-variables.md [hunk 8]
```diff
@@ -920,9 +920,9 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
+- 自 v6.1.0 起，TiDB 的 [Join Reorder 算法](/join-reorder.md) 开始支持 Outer Join。该变量用于控制这个支持行为，默认关闭，即不启用 Outer Join 的 Join Reorder。
+- 对于从旧版本升级上来的集群，该变量的默认值也会是 `OFF`。
```
Comment (TomShawn) [inline system-variables.md:925]:
```suggestion
- 对于从 v6.1.0 之前版本升级到 v6.1.0 及之后的版本，该变量的默认值为 `OFF`。对于从 v6.1.0 版本升级到之后的版本，该变量默认值为 `ON`。
```

**Review by AilinKid — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**